genisys/make_password.py

```diff
@@ -15,6 +15,11 @@ def make_password(self, raw_password, filename):
         secret_password = pbkdf2_sha256.hash(raw_password)
         self.config["users"]["root-login"] = secret_password
 
+        # store updated configuration back into config file
+        parser = YAMLParser
+        config = parser.as_dict(filename)
+        config["users"]["password"] = secret_password
+
```

Comment:
Remember that `config` is just a representation of the contents of the file, not the file itself. You still need some code to write the modified contents back to the original file. You'll have to check out the documentation for PyYaml on what methods it takes to do that.